genon/preprocessor/facade/attachment_processor.py

```
@@ -99,6 +99,41 @@
+    @wraps(func)
+    async def wrapper(*args, **kwargs):
+        result = await func(*args, **kwargs)
+
+        for r in result:
+            url = f"{GENOS_URL}/api/gateway/workflow/{GUARDRAIL_WORKFLOW_ID}"
+            headers = dict(Authorization=f"Bearer {GUARDRAIL_BEARER_TOKEN}")
+
+            if hasattr(r, "text"):
+                body = {"question": r.text}
+
+                res = requests.post(f"{url}/run/v2", json=body, headers=headers)
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)